source/gui/addonGui.py

```diff
@@ -303,16 +303,18 @@ def getAddonStatus(self, addon):
 			# Translators: The status shown for a newly installed addon before NVDA is restarted.
 			statusList.append(_("Install"))
 		# in some cases an addon can be expected to be disabled after install, so we want "install" to take precedence here
-		elif globalVars.appArgs.disableAddons or addon.isDisabled:
+		elif addon.isDisabled or globalVars.appArgs.disableAddons:

```

Comment:
Why did you make this change? note that if you touched the disabled property as suggested above, you can change this one as well.

Reply:
Same answer as before: `isDisabled` is meant to reflect the user setting. But I added a comment, to be clearer.